dojo/engine_tools/helpers.py

```diff
@@ -102,20 +113,20 @@ def expire_finding_exclusion(expired_fex: FindingExclusion) -> None:
             expired_fex.save()
             system_user = get_user(settings.SYSTEM_USER)
             logger.info(f"Expired finding exclusion: {expired_fex}")
-            note = get_note(system_user, "Finding has been removed from the whitelist as it has expired.")
+            note = get_note(system_user, f"Finding has been removed from the {expired_fex.type} as it has expired.")
             
             findings = Finding.objects.filter(
                 cve=expired_fex.unique_id_from_tool,
                 active=False,
```

Comment:
handle the status, because when is black_list the status is True and white_list is Flase